### PR TITLE
[W5.11][W12-1]Lim Jit Wei

### PR DESCRIPTION
**Pull request by jitwei98**
Added a sort command that sorts the underlying list by lexicographical order of the names

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -91,6 +91,17 @@ Returns `John Doe` but not `john`.
 * `find Betsy Tim John` +
 Returns Any person having names `Betsy`, `Tim`, or `John`.
```
Comment (sylchw) [inline docs/UserGuide.adoc:93]: User Guide is clear

File: src/seedu/addressbook/commands/HelpCommand.java [hunk 2]
```diff
@@ -17,6 +17,7 @@ public HelpCommand() {}
     public CommandResult execute() {
         return new CommandResult(
                 AddCommand.MESSAGE_USAGE
+                + "\n" + SortCommand.MESSAGE_USAGE
```
Comment (sylchw) [inline src/seedu/addressbook/commands/HelpCommand.java:20]: Necessary for help command to display message usage for sort

**Review by sylchw — APPROVED** [review]
Looks great

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -91,6 +91,17 @@ Returns `John Doe` but not `john`.
...
 
+== Sort the AddressBook : `sort`
```
Comment (okkhoy) [inline docs/UserGuide.adoc:94]: 👍 for updating the user guide

File: src/seedu/addressbook/data/AddressBook.java [hunk 4]
```diff
@@ -54,6 +54,13 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
         allPersons.remove(toRemove);
     }
 
+    /**
+     * Sort all persons by name alphabetically
```
Comment (okkhoy) [inline src/seedu/addressbook/data/AddressBook.java:58]: Remember the format for header comments given in the coding standards.

**Review by okkhoy — APPROVED** [review]
Good enhancement!

**Comment by okkhoy** [comment]
@jitwei98 
Some comments added. Please take note and close the pull request.

**Pull request closed without merging**